graph/src/env/mod.rs

```diff
@@ -211,6 +211,16 @@ impl EnvVars {
         let mapping_handlers = InnerMappingHandlers::init_from_env()?.into();
         let store = InnerStore::init_from_env()?.into();
 
+        // The default reorganization (reorg) threshold is set to 250.
+        // For testing purposes, we need to set this threshold to 0 because:
+        // 1. Many tests involve reverting blocks.
+        // 2. Blocks cannot be reverted below the reorg threshold.
+        // Therefore, during tests, we want to set the reorg threshold to 0.
+        let reorg_threshold =
+            inner
+                .reorg_threshold
+                .unwrap_or_else(|| if cfg!(debug_assertions) { 0 } else { 250 });
```

Comment:
Oh wow .. I didn't know you could use `cfg!` as a normal macro .. nice

Reply:
One thought: will tests still fail if `reorg_trheshold` is set to 1? It seems a little weird to use 0 here, but if that's what's needed to let tests pass, I am ok with it.

Reply:
Unfortunately no, there are some tests that fail when we se 1.